### ISSUE TITLE
Error E0A046E4

### ISSUE THREAD

**Issue by VersedXth** [MarcuzD/OCDM-old#1]
Im getting this error . It says something about 
```Searching for Mario Maker 3ds extdata``` 
I'm using the EU digital version 
![Report](https://user-images.githubusercontent.com/16269139/33638435-875acadc-da26-11e7-86d3-7fb5d7c4deca.jpg)

**Comment by MarcuzD** [comment]
> "ResultNotInitialized"
> using the digital version

did you ever open the game and got to the Coursebot screen? OCDM is not able to create a save 100% from scratch
also, what entrypoint are you using? cia? *hax? Rosalina?

**Comment by VersedXth** [comment]
Yes I did. I saved some of my levels with it and also some from Network. 
Using the cia

**Comment by RShadowhand** [comment]
I have the identical problem, however I found out it crashes with a different code if I take out my R4 cart from the device.

**Comment by VersedXth** [comment]
I'd tried that too 
without cart I get the Result:C880448D
The rest looks pretty much the same

**Comment by RShadowhand** [comment]
Exact same error code as mine, without the cart.

**Comment by MarcuzD** [comment]
> ResultMediaNotFound

it seems like I'd have to rewrite the save detection code, as I designed it to crash on purpose if no valid savefile can be opened... which means I failed at opening opening the on-SD savefile, which means that I failed at programming the save opener code... I'll see what I can do, but sadly I'll have to rewrite the save opener code from scratch...
